invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js

```diff
@@ -273,6 +263,39 @@ export class RDMDepositForm extends Component {
     header: { className: "inverted brand", style: { cursor: "pointer" } },
   };
 
+  /* TODO FIX ME and inject me */
+  PIDsProviders = [
+    {
+      scheme: "doi",
```

Comment:
Shall we only merge `DOI` since the other two were mainly a demo? Or we merge to be able to test, and remove before merging to master?

Reply:
Yes I can do that, but in any case we need to come back and fix the injection because now it is hardcoded.
I will do that, waiting for the answers below to update the code.

Reply:
Done!